app/views/caseload.html

```diff
@@ -12,133 +12,219 @@
 
 {% include "includes/sub-nav.html" %}
 
-{% if organisationLevel === 'team'%}
-<h2 class="heading-large">Caseload totals</h2>
+{% include "includes/export.html" %}
 
-<div id="caseloadSummary" class="grid-row">
+{% if organisationLevel !== 'offender-manager' %}
+  <h2 class="heading-large">Caseload totals</h2>
+  <div id="caseloadSummary" class="grid-row">              
   {% for caseType in caseloadDetails %}
     {% if caseType.displayName !== 'Overall' %}
-    <div class="column-one-third">
-      <div id="{{ caseType.displayName.toLowerCase() }}Total" class="data">
-        <span class="data-item bold-xlarge">{{ caseType.totalSummary }}</span>
-        <span class="data-item bold-xsmall">{{ caseType.displayName }} cases</span>
+      <div class="column-one-third">
+        <div id="{{ caseType.displayName.toLowerCase() }}Total" class="data">
+          <span class="data-item bold-xlarge">{{ caseType.totalSummary }}</span>
+          <span class="data-item bold-xsmall">{{ caseType.displayName }} cases</span>
+        </div>
       </div>
-    </div>
     {% endif %}
   {% endfor %}
-</div>
-{% endif %}
+  </div>
 
-{% include "includes/export.html" %}
```

Comment:
Export was above the tables as this is all that's exported, not the large summary values at the top. Any reason for moving to top of page?

Reply:
We need to have consistency on where the the `export` should be. In OM level the button is on the side of the table.

Reply:
There was an issue placing it inside the `{% if org !=='offender-manager'%}`.  I have no removed this and placed it underneathe the totals.  Think this is the best place for it as everything underneath it will be exported